tests/integration/cmd_pref_config_test.go

```diff
@@ -261,19 +261,19 @@ var _ = Describe("odo preference and config command tests", func() {
 	// 	It("should successfully set and unset variables", func() {
 	// 		//set env var
 	// 		helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
-	// 		helper.CmdShouldPass("odo", "create", "--s2i", "nodejs", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context)
-	// 		helper.CmdShouldPass("odo", "config", "set", "--now", "--env", "hello=world", "--context", commonVar.Context)
+	// 		helper.Cmd("odo", "create", "--s2i", "nodejs", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
+	// 		helper.Cmd("odo", "config", "set", "--now", "--env", "hello=world", "--context", commonVar.Context).ShouldPass()
 	// 		//*Check config
-	// 		configValue1 := helper.CmdShouldPass("odo", "config", "view", "--context", commonVar.Context)
+	// 		configValue1 := helper.Cmd("odo", "config", "view", "--context", commonVar.Context).ShouldPass().Out()
 	// 		helper.MatchAllInOutput(configValue1, []string{"hello", "world"})
 	// 		//*Check dc
 	// 		envs := oc.GetEnvsDevFileDeployment("nodejs", commonVar.Project)
 	// 		val, ok := envs["hello"]
 	// 		Expect(ok).To(BeTrue())
 	// 		Expect(val).To(ContainSubstring("world"))
 	// 		// unset a valid env var
-	// 		helper.CmdShouldPass("odo", "config", "unset", "--now", "--env", "hello", "--context", commonVar.Context)
-	// 		configValue2 := helper.CmdShouldPass("odo", "config", "view", "--context", commonVar.Context)
+	// 		helper.Cmd("odo", "config", "unset", "--now", "--env", "hello", "--context", commonVar.Context).ShouldPass()
+	// 		configValue2 := helper.Cmd("odo", "config", "view", "--context", commonVar.Context).ShouldPass()
```

Comment:
Missing .out as you are need to assign the output to configValue2

Reply:
https://github.com/openshift/odo/pull/4788#discussion_r651680858